docs/sources/installation/upgrading.md

```diff
@@ -117,3 +117,32 @@ One of the database migrations included in this release will update all annotati
 We've got one report where using systemd, PostgreSQL and a large amount of annotations (table size 1645mb) took 8-20 minutes for the database migration to complete. However, the grafana-server process was killed after 90 seconds by systemd. Any database migration queries in progress when systemd kills the grafana-server process continues to execute in database until finished.
 
 If you're using systemd and have a large amount of annotations consider temporary adjusting the systemd `TimeoutStartSec` setting to something high like `30m` before upgrading.
+
+## Upgrading to v6.0
+
+If you have text panels with script tags they will no longer work due to a new setting that per default disallow unsanitzied HTML.
+Read more [here](/installation/configuration/#disable-sanitize-html) about this new setting.
+
+### Authentication and security
```

Comment:
Might be a good idea to mention we use a new cookie name. `grafana_sess` in case people have some kind custom setup regarding cookies. Makes we question if we really need a new cookie name?

Reply:
Good point :+1: